.github/workflows/push-to-prod.yaml

```
@@ -57,34 +53,64 @@ jobs:
-        working-directory: ./terraform
+      - name: Set up Cloud SDK
+        uses: google-github-actions/setup-gcloud@v3
 
-      - name: Terraform Format
-        run: terraform fmt -check
-        working-directory: ./terraform
+      - id: 'deploy'
+        uses: 'google-github-actions/deploy-cloudrun@v3'
+        with:
+          service: 'terrastack-docs'
+          image: 'gcr.io/${{secrets.GCP_PROJECT_ID}}/terrastack-docs:${{ steps.timestamp.outputs.container-tag }}'
```

actionlint (1.7.11)

[error] property "timestamp" is not defined in object type {}

(expression)